families/supplychain/restapi/openapi.yaml

```diff
@@ -0,0 +1,378 @@
+        - $ref: "#/parameters/count"
+        - $ref: "#/parameters/min"
+        - $ref: "#/parameters/max"
+      responses:
+        200:
+          description: Successfully retrieved Applications
+          schema:
+            properties:
+              data:
+                type: array
+                items:
+                  $ref: "#/definitions/Application"
+              link:
+                $ref: "#/definitions/Link"
+              paging:
+                $ref: "#/definitions/Paging"
+        400:
+          $ref: "#/responses/400BadRequest"
+        500:
+          $ref: "#/responses/500ServerError"
+        503:
+          $ref: "#/responses/503ServiceUnavailable"
+  /applications/{record_id}:
+    parameters:
+      - $ref: "#/parameters/record_id"
+    get:
+      summary: Fetches a particular Application.
+      description: >
+        Fetches a paginated list of Applications for a record. Applications are
+        identified by the identifier of the Record they are associated with.
+        Using the `applicant` parameter will narrow the list to any
+        Applications that have a matching value in their applicant field. Using
+        the `state` parameter will narrow the list to any Applications that
+        have a matching value in their state field.
+      parameters:
+        - name: applicant
+          in: query
+          type: string
+          description: The public key of the applicatant Agent to filter by
+        - name: state
+          in: query
+          type: string
+          description: A application state to filter by
+            [OPEN|CANCELED|REJECTED|ACCEPTED]
+        - $ref: "#/parameters/count"
+        - $ref: "#/parameters/min"
+        - $ref: "#/parameters/max"
+      responses:
+        200:
+          description: Successfully fetched Applications
+          schema:
+            properties:
+              data:
+                type: array
+                items:
+                  $ref: "#/definitions/Application"
+              link:
+                $ref: "#/definitions/Link"
+              paging:
+                $ref: "#/definitions/Paging"
+        400:
+          $ref: "#/responses/400BadRequest"
+        404:
+          $ref: "#/responses/404NotFound"
+        500:
+          $ref: "#/responses/500ServerError"
+        503:
+          $ref: "#/responses/503ServiceUnavailable"
+  /records:
+    get:
+      summary: Fetches the data for the current set of Records
+      description: >
+        Fetches a paginated list of all Records. Using the `identifier` filter
+        parameter will narrow the list to any Records that have an identifier
+        containing the specified string.
+
+        Records are identified by their serial number or other marking on the
+        physical asset.
+      parameters:
+        - name: identifier
+          in: query
+          type: string
+          description: A partial identifier to filter Records by
```

Comment:
They could be anything, we are expecting the common case to serial numbers which could be a arbitrary identifier or a "smart number", if it is a "smart number" then a partial filter can be useful